core/src/main/scala/org/bitcoins/core/script/ScriptOperationFactory.scala

```diff
@@ -72,9 +81,19 @@ trait ScriptOperationFactory[T <: ScriptOperation] extends BitcoinSLogger {
 
 object ScriptOperation extends ScriptOperationFactory[ScriptOperation] {
 
-  val operations: Seq[ScriptOperation] = {
+  /** This contains duplicate operations
+   * There is an optimization here by moving popular opcodes
+   * to the front of the vector so when we iterate through it,
+   * we are more likely to find the op code we are looking for
+   * sooner */
+  final override val operations: Vector[ScriptOperation] = {
+    Vector(OP_DUP, OP_HASH160, OP_EQUALVERIFY, OP_CHECKSIG) ++ //p2pkh
+    Vector(BytesToPushOntoStack.push20Bytes, BytesToPushOntoStack.push33Bytes,
+      BytesToPushOntoStack.push32Bytes) ++ //popular push op codes
+    Vector(OP_HASH160, OP_EQUAL) ++ //p2sh
+    Vector(OP_0, OP_1, OP_CHECKMULTISIG) //multisig
     ScriptNumberOperation.operations ++
-      Seq(OP_FALSE, OP_PUSHDATA1, OP_PUSHDATA2, OP_PUSHDATA4, OP_TRUE) ++
+      Vector(OP_FALSE, OP_PUSHDATA1, OP_PUSHDATA2, OP_PUSHDATA4, OP_TRUE) ++
```

Comment:
`OP_FALSE` and `OP_TRUE` are the same as `OP_1` and `OP_0`, no? those are already included in the list